ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/MetadataUtil.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.tools.migrations.util;
+
+import io.confluent.ksql.api.client.BatchedQueryResult;
+import io.confluent.ksql.api.client.Client;
+import io.confluent.ksql.api.client.Row;
+import io.confluent.ksql.tools.migrations.MigrationConfig;
+import io.confluent.ksql.tools.migrations.MigrationException;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+public class MetadataUtil {
+  private MetadataUtil() {
+  }
+
+  public static String getCurrentVersion(final MigrationConfig config) {
```

Comment:
Since both `apply` and `validate` will be making multiple client calls, I think it'd be better to have this method accept the client as a param (and leave it up to the calling commands to handle properly closing the client). 

If you can think of use cases where the only client call is to get the current version, we can have a convenience overload that creates and closes the client as this implementation currently does, but primarily we should be using the other version (that supplies the client).